From 30931a8a1afa4669c94574bb7a80a22578911b1a Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Mon, 7 Mar 2016 20:45:11 +0100
Subject: [PATCH] Fall back to look for libudev.pc if udev.pc is not usable

Signed-off-by: Balint Reczey <balint@balintreczey.hu>
---
 src/libcec/cmake/CheckPlatformSupport.cmake | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/libcec/cmake/CheckPlatformSupport.cmake b/src/libcec/cmake/CheckPlatformSupport.cmake
index b8a16c8..0b5e19d 100644
--- a/src/libcec/cmake/CheckPlatformSupport.cmake
+++ b/src/libcec/cmake/CheckPlatformSupport.cmake
@@ -74,6 +74,10 @@ else()
 
   # udev
   pkg_check_modules(UDEV udev)
+  if (NOT UDEV_FOUND)
+    # fall back to finding libudev.pc
+    pkg_check_modules(UDEV libudev)
+  endif()
   if (UDEV_FOUND)
     set(HAVE_LIBUDEV 1)
     set(LIB_INFO "${LIB_INFO}, P8_detect")
-- 
2.1.4

