From: Balint Reczey <balint@balintreczey.hu>
Date: Fri, 7 Aug 2015 00:12:37 +0200
Subject: Fix libdir in pkgconfig file

Forwarded: not-needed
---
 src/libcec/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libcec/CMakeLists.txt b/src/libcec/CMakeLists.txt
index 15b2ef8..e258f0d 100644
--- a/src/libcec/CMakeLists.txt
+++ b/src/libcec/CMakeLists.txt
@@ -171,7 +171,7 @@ else()
   configure_pc_file(cec ${CMAKE_CURRENT_SOURCE_DIR}/libcec.pc.in
                         ${CMAKE_CURRENT_SOURCE_DIR}/libcec.pc
                         ${CMAKE_INSTALL_PREFIX}
-                        ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}
+                        ${CMAKE_INSTALL_LIBDIR}
                         ${CMAKE_INSTALL_PREFIX}/include)
 
   install(FILES         ${CMAKE_CURRENT_SOURCE_DIR}/libcec.pc
